rpcs3/Emu/System.cpp

```diff
@@ -931,7 +931,7 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 					break;
 				}
 
-				if (fs::file sfb_file{parent_dir + "/PS3_DISC.SFB"}; sfb_file && !sfb_file.stat().is_directory && sfb_file.size() >= 4 && sfb_file.read<u32>() == ".SFB"_u32)
+				if (fs::file sfb_file{parent_dir + "/PS3_DISC.SFB"}; sfb_file && sfb_file.size() >= 4 && sfb_file.read<u32>() == ".SFB"_u32)
 				{
```

Comment:
What was wrong with this line?

Reply:
I wasn't sure if it was related to the other changes and I decided to rather revert too much than too little and then have to do another revert. We can introduce the changes back in future PRs, its not a problem.